ui/app/AppLayouts/Profile/Sections/SyncContainer.qml

```diff
@@ -18,173 +18,200 @@ Item {
         }
     }
 
-    Item {
+    SyncHistoryModal {
```

Comment:
Maybe we should put this in a `Component` and open it using `openPopup`  so it is not created on login.